apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationPreviewList.tsx

```diff
@@ -0,0 +1,186 @@
+import { useIntl } from "react-intl";
+import PencilSquareIcon from "@heroicons/react/24/outline/PencilSquareIcon";
+
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { PreviewList, PreviewMetaData, Well } from "@gc-digital-talent/ui";
+
+import { getClassificationName } from "~/utils/poolUtils";
+import {
+  applicationStatus,
+  getApplicationStatusChip,
+} from "~/utils/poolCandidate";
+import useRoutes from "~/hooks/useRoutes";
+
+import { ApplicationDate } from "./MetadataDate";
+import ReviewApplicationDialog from "./ReviewApplicationDialog";
+
+const ReviewApplicationPreviewList_Fragment = graphql(/* GraphQL */ `
+  fragment ReviewApplicationPreviewList on PoolCandidate {
+    ...ReviewApplicationDialog
+    id
+    finalDecisionAt
+    submittedAt
+    removedAt
+    finalDecision {
+      value
+    }
+    assessmentStatus {
+      assessmentStepStatuses {
+        step
+      }
+      overallAssessmentStatus
+      currentStep
+    }
+    pool {
+      id
+      name {
+        localized
+      }
+      classification {
+        group
+        level
+        minSalary
+        maxSalary
+      }
+      closingDate
+      areaOfSelection {
+        value
+      }
+      screeningQuestions {
+        id
+      }
+    }
+  }
+`);
+
+interface ReviewApplicationPreviewListProps {
+  applicationsQuery: FragmentType<
+    typeof ReviewApplicationPreviewList_Fragment
+  >[];
+}
+
+const ReviewApplicationPreviewList = ({
+  applicationsQuery,
+}: ReviewApplicationPreviewListProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
+  const applications = getFragment(
+    ReviewApplicationPreviewList_Fragment,
+    applicationsQuery,
+  );
+
+  return (
+    <>
+      {applications.length ? (
+        <PreviewList.Root>
+          {applications.map((application) => {
+            const { id, pool, submittedAt, finalDecisionAt } = application;
+
+            const status = getApplicationStatusChip(
+              application.submittedAt,
+              pool.closingDate,
+              application.removedAt,
+              application.finalDecisionAt,
+              application.finalDecision?.value,
+              pool.areaOfSelection?.value,
+              application.assessmentStatus,
+              pool.screeningQuestions,
+              intl,
+            );
+
+            const applicationMetadata: PreviewMetaData[] = [
+              {
+                key: "status",
+                type: "chip",
+                color: status.color,
+                children: status.label,
+              },
+              {
+                key: "classification",
+                type: "text",
+                children: pool?.classification
+                  ? getClassificationName(pool?.classification, intl)
```

Comment:
We need to add something in here for abbreviations. Right now it reads as the word "it" for IT instead of I-T separately.

Reply:
Is this the only page it happens on? Because we use this function on a few other pages (e.g. JobPosteTemplate) I just want to know if I have a unique case or not.

Reply:
I will move this to the polish ticket

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/issues/12699